
uint8   GEAR_DRIVE=11
uint8   GEAR_NEUTRAL=12
uint8   GEAR_REVERSE=13

bool    ready
bool    driverless
uint8   gear
bool    hand_brake
bool    emergency_brake
float32 speed
float32 roadwheel_angle

